src/app/speakers/speakers/speakers.component.scss

```diff
@@ -50,16 +50,21 @@
   place-items: stretch;
   place-content: stretch;
   margin-bottom: 20px;
+
   @media (min-width: $desktop-min-size) {
     grid-gap: 15px;
-    grid-template-columns: repeat(3, 1fr);
+    grid-template-columns: repeat(2, 1fr);
```

Comment:
If only 2 columns it's probably overkill to use repeat, maybe `1fr 1fr` should be shorter.

Reply:
true, fixing in next PR